routes/leaguesearch.js

```diff
@@ -109,57 +126,6 @@ router.get('/:league', function(req, res) {
 })
 
 
-var createTeamLists = function(teamobject){
```

Comment:
Yea so I had originally been returning the top 10 teams for each league (so up to 50 teams..) to the leaguesearch/ page
but-- it seems like we just want to display the top 10 regardless of league there.. so I removed this function and did that. You'll see that page now has top 10 with a mix of nba and nhl  (compared to the leaguesearch/nba page that has top 10 only nba teams)